core/arch/aarch64/emit_utils.c

```diff
@@ -199,7 +199,8 @@ insert_exit_stub_other_flags(dcontext_t *dcontext, fragment_t *f, linkstub_t *l,
         /* mov x0, ... */
         pc = insert_mov_imm(pc, DR_REG_X0, (ptr_int_t)l);
         num_nops_needed = 4 - (pc - write_stub_pc - 1);
-        uint target_pc_slot_offs = (uint *)get_target_pc_slot(f, stub_pc) - pc;
+        uint *target_pc_slot = (uint *)get_target_pc_slot(f, stub_pc);
```

Comment:
The `uint *` type is confusing: the pc is 8 bytes, not 4.  It makes this look like a bug where it assumes the pc fits in a uint.

Reply:
Made it a `ptr_uint_t*` type.